src/components/ResourceBrowser/ResourceList/ResourceList.tsx

```diff
@@ -369,7 +369,7 @@ export default function ResourceList() {
             type: 'start',
             payload: {
                 searchText: _searchText,
-                list: _resourceList.data,
+                list: _resourceList.data || [],
```

Comment:
we can add check of _resourceList on line no 367 may be

Reply:
done